docs/breaking.md

```diff
@@ -7,22 +7,34 @@
 
 # Proposed Breaking Changes
 
-## Oct 25, 2019
+## Log
+
+### Nov 5, 2019
+
+- Route changes will not be pluralized, instead they will be all singular
+- `workflows` will be removed from the [Get a pipeline](https://circleci.com/docs/api/v2/#get-a-pipeline) response. This information will now be retrieved in two new endpoints: get workflows by pipeline and get scheduled workflows by project
```

Comment:
~ these two routes will be `GET /pipeline/{pipeline-id}/workflow` and `GET /project/{project-slug}/scheduled-workflow` wdyt about mentioning those here?

Reply:
sgtm